### PR TITLE
feat: about grid 

### PR DESCRIPTION
**Pull request by DanielGovrin**

### CODE REVIEW

File: _codux/boards/about-grid.board.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    grid-column: 1 / 4;
+    @media (width < 768px) {
+        grid-row: 4 / 4;
+        grid-column: 1 / 4;
+    }
+    @media (width < 480px) {
+        grid-row: 6;
+        grid-column: 1;
+    }
+    max-height: 160px;
```
Comment (0xRezaa) [inline _codux/boards/about-grid.board.css]: please move this above the media queries

**Review by 0xRezaa — COMMENTED** [review]
All of the boards I have checked in this repo use camelCase convention for the classes. You were considering using the BEM convention. What is the convention in the repo?

**Comment by 0xRezaa** [comment]
https://github.com/user-attachments/assets/d8d9f03e-5e40-4f7e-8611-7e8bd8102ea7

**Review by 0xRezaa — APPROVED** [review]
lgtm other than 2 small comments

**Comment by 0xRezaa** [comment]
Uploading Screen Recording 2024-11-03 at 18.04.38.mov…

**Comment by 0xRezaa** [comment]
Just a recording of me reviewing it visually
https://github.com/user-attachments/assets/c435b38d-500d-435c-a4ad-0da2b5f849f2

**Comment by DanielGovrin** [comment]
I used dashes to separate between words and the board name as an initial to the class name

**Pull request merged**